yesql.js

```diff
@@ -25,6 +25,11 @@ const readSqlFiles = (dir, options = {}) => {
 
 const pg = (query, options = {}) => {
   return (data = {}) => {
+
+    // sanitize comments
+    query = query.replace(/--[^\n]*$/gm, '')
```

Comment:
Is the `[^\n]` needed as it ends with `$`?
Also is the `m` required in the end?

I took those out in #27 and seems to work with the test cases there.